vm/src/vm.rs

```diff
@@ -207,6 +207,12 @@ impl VirtualMachine {
         vm
     }
 
+    /// `update_sysmodule` should be called when the `vm.stdlib_inits` are changed,
+    /// and then the `builtin_module_names` of `sys` module will be updated.
+    pub fn update_sysmodule(&self) {
+        sysmodule::make_module(self, self.sys_module.clone(), self.builtins.clone());
```

Comment:
I feel like this might have unintended side effects, it might be better to explicitly update `builtin_module_names` than call all of `sysmodule::make_module`

Reply:
Maybe we can define a stage `initialize` after VM created.
And put `init_importlib`, `sysmodule::make_module` and `builtins::make_module` in `initialize`.  Before `initialize` the `stdlib_inits` can be changed base on different environments.